components/automate-cli/pkg/verifyserver/services/softwareversionservice/softwareversionservice.go

```diff
@@ -186,19 +187,31 @@ func (sv *SoftwareVersionService) checkKernelVersion(kernelFilePath, osName stri
 		return failureResponse(KERNAL_VERSION_CHECK, "Its not feasible to determine the Kernal version of the system", "Please run automate on the supported platforms."), err
```

Comment:
check the spelling of "Kernel" and update everywhere

Reply:
updated